repositories.bzl

```diff
@@ -22,9 +22,10 @@ airy_jvm_deps = [
     "javax.activation:javax.activation-api:1.2.0",
     "javax.validation:validation-api:2.0.1.Final",
     "javax.xml.bind:jaxb-api:2.3.1",
-    "org.apache.logging.log4j:log4j-core:2.12.1",
-    "org.apache.logging.log4j:log4j-slf4j-impl:2.12.1",
-    "org.slf4j:slf4j-api:1.7.29",
+    "org.apache.logging.log4j:log4j-core:2.15.0",
+    "org.apache.logging.log4j:log4j-slf4j-impl:2.15.0",
+    "org.slf4j:slf4j-nop:1.7.32",
```

Comment:
This is to remove the [following warning](http://www.slf4j.org/codes.html#StaticLoggerBinder)

Reply:
Very cool. You still need to add it here though so that it shows up in the class path for users of the log library: https://github.com/airyhq/airy/blob/a546e91c0905224cdee871dadd8a9c8ef94fa42b/lib/java/log/BUILD#L9

Reply:
I just did but the warning still there. For example while trying to build the webhook publisher. I still get the warnings as follow
```sh
$ bazel build //backend/webhook/publisher:webhook-publisher
INFO: Analyzed target //backend/webhook/publisher:webhook-publisher (284 packages loaded, 2338 targets configured).
INFO: Found 1 target...
INFO: From generating avro srcs:
Input files to compile:
  backend/avro/tag.avsc
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
INFO: From generating avro srcs:
Input files to compile:
  backend/avro/http-log.avsc
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
INFO: From generating avro srcs:
Input files to compile:
  backend/avro/metadata.avsc
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
INFO: From generating avro srcs:
Input files to compile:
  backend/avro/webhook.avsc
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
INFO: From generating avro srcs:
Input files to compile:
  backend/avro/channel.avsc
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
INFO: From generating avro srcs:
Input files to compile:
  backend/avro/message.avsc
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Target //backend/webhook/publisher:webhook-publisher up-to-date:
  bazel-bin/backend/webhook/publisher/app.jar
  bazel-bin/backend/webhook/publisher/app
  bazel-bin/backend/webhook/publisher/MANIFEST.MF
  bazel-bin/backend/webhook/publisher/app_springboot.jar
INFO: Elapsed time: 19.415s, Critical Path: 13.51s
INFO: 296 processes: 2 internal, 262 darwin-sandbox, 32 worker.
INFO: Build completed successfully, 296 total actions
```

I try to add `@maven//:org_slf4j_slf4j_nop` and `//lib/java/log` to the *app_deps* in _backend/webhook/publisher/BUILD_ and the result is the same. Same warnings. Any ideas why `nop` is not being inserted into the class path?

Reply:
@juan-sebastian For me it works without the warning when I check out your PR. I'll approve now since the log4j update is already there.